fix(bootstrapper): Check module function configure before execute it

pnizer opened the pull request:
Prevent error 'TypeError: customConfig.configure it not a function' when a module is set by <... aurelia-app="my-module"> and my-module.js does not have exported a configure function.

pnizer committed:
fix(bootstrapper): Check module function configure before execute it

Prevent error 'TypeError: customConfig.configure it not a function' when a module is set by <... aurelia-app="my-module"> and my-module.js does not have exported a configure function.

EisenbergEffect commented:
Can you describe how you are using this without the configure function? I'd like to understand your use case and approach before merging. Thanks!

pnizer commented:
The application does not work without the configure function, but the error in console does not help someone to understand that the error is the absence of configure function. Maybe the real 'solution' is throwing an exception with a message explaining the problem to developer.

EisenbergEffect commented on src/index.js:
```diff
@@ -89,7 +89,7 @@ function config(loader, appHost, configModuleId) {
 
   if (configModuleId) {
-    return loader.loadModule(configModuleId).then(customConfig => customConfig.configure(aurelia));
+    return loader.loadModule(configModuleId).then(customConfig => customConfig.configure && customConfig.configure(aurelia));
```
I definitely think that throwing a descriptive error if the `configure` method is missing would be better than failing silently. Would you be willing to update this PR with that change?

EisenbergEffect reviewed (changes requested):
Can we throw an error instead of failing silently?

pnizer committed:
Module must declare a configure function

Throw an exception with descriptive error when a module without configure function is used

pnizer commented:
Throwing a descriptive error when there is no configure function.
